src/net_processing.cpp

```diff
@@ -1771,7 +1761,7 @@ void static ProcessGetData(CNode& pfrom, const CChainParams& chainparams, CConnm
     // expensive to process.
     if (it != pfrom.vRecvGetData.end() && !pfrom.fPauseSend) {
         const CInv &inv = *it++;
-        if (inv.type == MSG_BLOCK || inv.type == MSG_FILTERED_BLOCK || inv.type == MSG_CMPCT_BLOCK || inv.type == MSG_WITNESS_BLOCK) {
+        if (inv.IsGenBlkMsg()) {
```

Comment:
Could this result in unintended behavioral change?

Previously if `inv.type == MSG_FILTERED_WITNESS_BLOCK` we would not execute the code inside this `if` block. With this patch we would execute it.

Related to https://github.com/bitcoin/bitcoin/pull/19610#discussion_r465175043

Reply:
That message type is unused (see https://github.com/bitcoin/bitcoin/pull/19610#discussion_r468510147) and has since been commented out by #19721 (merged).

Reply:
Removing it in the rebase.